docs/DeveloperGuide.md

```diff
@@ -1,38 +1,152 @@
 # Developer Guide
 
 ## Acknowledgements
-
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* Java Coding Standard (Basic): [Official Documentation](https://se-education.org/guides/conventions/java/basic.html)
+* Code Quality (Guide): (https://nus-cs2113-ay2425s1.github.io/website/se-book-adapted/chapters/codeQuality.html)
 
 ## Design & implementation
-
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### "Add" Command Feature
+The "AddCommand" is responsible for adding fields and records to the inventory. 
+Concurrently, "add" command is implemented with behaviour to update CSV file to persist changes made to inventory.
+This command supports the following options:
+* '-h': Add custom fields to the inventory.
+* '-d': Add records to the inventory corresponding to defined fields.
+* '-hu': Updates fields in the inventory.
```

Comment:
You may consider including an example of how the command looks like in this section